docs/user-guide/simulator_integration.md

```diff
@@ -211,7 +211,7 @@ h.helicsPublicationPublishChar(pub, string_value)
 For sending a message through an endpoint, that once again looks a little bit different, in this case because - unlike with a publication - a message requires a destination. If a default destination was set when the endpoint was registered (either through the config file or through calling `h.helicsEndpointSetDefaultDestination()`), then an empty string can be passed. Otherwise, the destination must be provided as shown in API call below where dest is the destination and msg is the message to be sent.
 
 ```python
-h.helicsEndpointSendMessageRaw(end, dest, msg)
+h.helicsEndpointSendTo(end, msg, dest)
```

Comment:
Even here, I think `helicsEndpointSendBytesTo` reads better than `helicsEndpointSendTo`.

Reply:
I don't really like it there since there is no alternative,  in the other cases Bytes are used there are other functions for the different types.

Reply:
I think just by reading the function signature (especially in languages like Python, Julia, etc):

```
h.helicsEndpointSendTo(end, msg, dest)
```

it can be hard to tell whether this takes a String, Bytes or a Message object.